indexer/src/main.rs

```diff
@@ -0,0 +1,186 @@
+use clap::Parser;
+
+pub use cached::SizedCache;
+use futures::future::try_join_all;
+use futures::{try_join, StreamExt};
+use tokio::sync::Mutex;
+use tracing::{debug, info};
+
+use explorer_database::{adapters, models, receipts_cache};
+
+use crate::configs::Opts;
+
+mod configs;
+mod metrics;
+
+// Categories for logging
+const INDEXER_FOR_EXPLORER: &str = "indexer_for_explorer";
```

Comment:
Same, let's get rid of `explorer`?

Reply:
but it is the name of the project. Let's keep it as is, please :)